src/containers/FacilityForm.js

```diff
@@ -74,6 +78,49 @@ function FacilityForm({ backend, history }) {
     );
   }
 
+  const formFieldsData = [
```

Comment:
I considered putting this in a data file (eg: `data/forms/facilityForm`), but we would need to pass a number of values to that data file that are available for free in this file (eg: `customOnChange` functions and `value`). That's not an impossible task if people have strong feelings.